### DISCUSSION TITLE
Configuring Podman for Quarkus Dev Services and Testcontainers on Linux

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#21955] [Quarkus Blog/Website]
# Configuring Podman for Quarkus Dev Services and Testcontainers on Linux

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-devservices-testcontainers-podman/

**Comment by agreedSkiing** [comment]
Would recommend to change the export to use the instructions from https://github.com/containers/podman/blob/main/docs/tutorials/socket_activation.md#socket-activation-of-the-api-service to set the correct podman socket
